src/boot/efi/secure-boot.c

```diff
@@ -117,6 +117,7 @@ EFI_STATUS secure_boot_enroll_at(EFI_FILE *root_dir, const char16_t *path, bool
                 }
         }
 
+        printf("Cold-resetting the system now!");
```

Comment:
Let's not use confusing language here. Also, if the safety countdown happened, the message is not shown on it's own line.
```suggestion
        printf("Rebooting the system now!\n");
```

Reply:
Maybe even say `Custom SecureBoot keys successfully enrolled, rebooting the system now!`

Reply:
Done, thanks for the review!